app/src/main/java/infrmr/newsapp/github/com/ifrmr/ArticleListFragment.java

```diff
@@ -265,6 +324,8 @@ private InputStream downloadUrl(String urlString) throws IOException {
      * Method for updating article data after successful download
      */
     void onItemsLoadComplete(ArrayList<TheVergeXmlParser.Entry> downloadedArticles) {
+        Log.i("UTD", "onLoadComplete: isUpToDate = true");
+        MainActivity.isUpToDate = true;
```

Comment:
Not sure if this boolean check is the best way to go about this...will need to look at it in more detail